sys-apps/arch-chroot/metadata.xml

```diff
@@ -5,6 +5,10 @@
       <email>cat@catcream.org</email>
       <name>Alfred Persson</name>
     </maintainer>
+	<maintainer type="person" proxied="yes">
```

Comment:
Fix indentation

Reply:
yeah saw it as I pushed and then I lost internet connection

Reply:
Prayed you didn't see before I got it back

Reply:
🤣